menu_folder: "/home/user/crystalfontz_scripts"
